backend/app/routes/game_routes.py

```
@@ -145,9 +145,22 @@ def get_available_levels(current_level):
     weights = [0.3 if l < current_level else 0.7 for l in levels]
     return levels, weights
 
-@bp.route('/generate-question/<int:user_id>/<int:level>', methods=['GET'])
-def generate_question(user_id, level):
+@bp.route('/generate-question/<int:level>', methods=['GET'])
+def generate_question(level):
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_question`

(ANN201)